code/integration-tests/runtime-tests/tests/utils/assets.json

```diff
@@ -0,0 +1,38 @@
+{
+  "pica": {
```

Comment:
do not use map. 

when you send, you know receiver, approximate amount - fees. just observe event, and transfer futher.

why? because we can change in devnet these numbers and denoms, and names, as it was several times.

so need only to know 1 assets which is sent, but all other assets observer on other chains as event with well know recever and amount (minus 1/10 for fees).

Reply:
alternative, I can encode it like here:

https://github.com/ComposableFi/composable/blob/main/code/xcvm/cvm.json

see that one may build graph of assets per chains.

it just lacks composable/picass - I can add fast if you are with this approach.

why?

because not doing so will lead to brittle tests hard to maintain.

Reply:
I moved all these into config json file but still kept the asset info for only ksm. Currently tests compare total isssuances, escrow balances and charged fee. For that I need to be able to query the balances before the tx actually arrive on the chain. But I moved all this logic into one json file. So if there is a need to modify, or add more assets, I think this can be done here easily.